.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["react", "es2015", "stage-0"]
```

Comment:
I'll try to see which ones supports template string methods e.g. `Relay.QL\`...\``